iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractLoadingLimits.java

```diff
@@ -85,6 +95,58 @@ public L setPermanentLimit(double permanentLimit) {
         return (L) this;
     }
 
+    @Override
+    public L setTemporaryLimitValue(int acceptableDuration, double temporaryLimitValue) {
+        if (temporaryLimitValue < 0 || Double.isNaN(temporaryLimitValue)) {
+            throw new ValidationException(validable, "Temporary limit value must be a non-negative double");
+        }
+
+        // Identify the limit that needs to be modified
+        TemporaryLimit identifiedLimit = getTemporaryLimit(acceptableDuration);
+        if (identifiedLimit == null) {
+            throw new ValidationException(validable, "No temporary limit found for the given acceptable duration");
+        }
+
+        TreeMap<Integer, TemporaryLimit> temporaryLimitTreeMap = new TreeMap<>(this.temporaryLimits);
+        // Creation of index markers
+        Map.Entry<Integer, TemporaryLimit> biggerDurationEntry = temporaryLimitTreeMap.lowerEntry(acceptableDuration);
+        Map.Entry<Integer, TemporaryLimit> smallerDurationEntry = temporaryLimitTreeMap.higherEntry(acceptableDuration);
+
+        double oldValue = identifiedLimit.getValue();
+
+        if (isTemporaryLimitValueValid(biggerDurationEntry, smallerDurationEntry, acceptableDuration, temporaryLimitValue)) {
+            LOGGER.info("Temporary limit value changed from {} to {}", oldValue, temporaryLimitValue);
+        } else {
+            LOGGER.warn("Temporary limit value changed from {} to {}, but it is not valid", oldValue, temporaryLimitValue);
```

Comment:
Can you add `group.getValidable().getMessageHeader()` as a prefix in the logged messages?